draft-ietf-masque-h3-datagram.md

```diff
@@ -1113,15 +512,12 @@ STREAM(44): HEADERS            -------->
   :method = webtransport
   :path = /hello
   :authority = webtransport.example.org:443
-  Origin = https://www.example.org:443
-
-STREAM(44): DATA                -------->
-  Capsule Type = REGISTER_DATAGRAM
-  Datagram Format Type = WEBTRANSPORT_DATAGRAM
-  Datagram Format Additional Data = ""
+  origin = https://www.example.org:443
+  sec-capsule-protocol = ?1
```

Comment:
It would be good to have an example that uses capsules but does not include this header field.

Reply:
Agreed

Reply:
I removed the header from the WebTransport example.